### PR TITLE
Iteration3 al

### PR DESCRIPTION
**Pull request by alex-latham**
- Created team_stats class and test file
- Switched Switch game_id, season, away_team_id, home_team_id, team_id, and franchiseid from strings to integers
- Correct camelcase symbol names

### CODE REVIEW

File: lib/team.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class Team
   attr_reader :team_id,
-              :franchiseId,
-              :teamName
+              :franchiseid,
```
Comment (whitneykidd) [inline lib/team.rb:3]: Nice catch on changing the casing here

File: lib/team.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 class Team
   attr_reader :team_id,
-              :franchiseId,
```
Comment (tylertomlinson) [inline lib/team.rb:3]: WHO DID THIS ORIGINALLY!? WOW. Good catch

Comment (alex-latham) [reply lib/team.rb:3]: It was a team effort

File: test/game_stats_test.rb [hunk 3]
```diff
@@ -56,8 +56,8 @@ def test_it_can_calculate_average_goals_per_game
   end
 
   def test_it_can_calculate_average_goals_per_season
-    goals_by_season = { '20152016' => 5.33, '20132014' => 5, '20142015' => 3,
-                        '20162017' => 4}
+    goals_by_season = { 20152016 => 5.33, 20132014 => 5, 20142015 => 3,
```
Comment (Maxwell-Baird) [inline test/game_stats_test.rb:59]: good job on making this numbers instead of strings

Comment (alex-latham) [reply test/game_stats_test.rb:59]: Thanks!

**Pull request merged**